src/pages/index.jsx

```diff
@@ -1,60 +1,28 @@
+/* Initial redirect to '/de' or 'en' should happen, so we don't have to keep this file up to date */
```

Comment:
Sollten am besten per `.htaccess` direkt auf `/de` o. `/en` redirecten, wenn man die Gatsby-Page per `/` aufruft.
Momentan wird hier in der Einstiegs-Page von Gatsby erstmal die deutsche Lokalisierung genutzt. Leider mit Code-Duplikat und wo die Filterung und das Mapping der Grafiken für die Übersichtsseite nicht zur Build-Phase stattfindet.